kloppy/infra/serializers/event/opta/deserializer.py

```diff
@@ -226,16 +229,13 @@ def _parse_f24_datetime(dt_str: str) -> float:
 
 def _parse_pass(raw_qualifiers: Dict[int, str], outcome: int) -> Dict:
     if outcome:
-        receiver_coordinates = Point(
-            x=float(raw_qualifiers[140]), y=float(raw_qualifiers[141])
-        )
         result = PassResult.COMPLETE
     else:
         result = PassResult.INCOMPLETE
-        # receiver_coordinates = None
-        receiver_coordinates = Point(
-            x=float(raw_qualifiers[140]), y=float(raw_qualifiers[141])
-        )
+    receiver_coordinates = Point(
```

Comment:
What is the rationale behind using (0,0) as the default location for the receiver? I'd rather make it explicit to the user that the location of the receiver is unknown. We could perhaps consider implementing the _Null Object_ pattern for the `Point` class to deal with situations where locations are missing.

Reply:
Only one event, in one match in over a thousand analysed Opta F 24 files, had missing receiver coordinates. Therefore, I went for the quick and dirty solution, (0,0) as the default location, that at least allowed loading in the data.

But would of course be fine with the null object pattern for Point class, I'll have a look.

Reply:
I'm fine with sticking to your current approach if missing receiver coordinates are rare.